test/runtests.jl

```diff
@@ -21,6 +21,7 @@ end
 
 # This makes sure that the values are within expected ranges
 @testset "Validation" begin
+    @test length(_currency_data) == 155
```

Comment:
This should definitely not be there. You might want to test that there are at least 155, because I doubt that a currency would be removed from the table, but not that it is precisely 155.